specification/cosmos-db/resource-manager/Microsoft.DocumentDB/DocumentDB/preview/2025-11-01-preview/softdeleted.json

```diff
@@ -0,0 +1,775 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB",
+    "description": "Azure Cosmos DB Database Service Resource Provider REST API",
+    "version": "2025-04-15"
```

Comment:
>"2025-04-15" [](http://example.com/codeflow?start=15&length=12)

Please create a copy of files and folders from last known api version and push them and then make changes to them for the new api version. That makes it easier to review changes rather than looking at all files that shows as newly added ones